README.rst

```diff
@@ -326,6 +326,11 @@ Choose from the list of available fixers:
                         Remove trailing commas in list
                         function calls.
 
+* **method_separation** [symfony]
+                        Methods in a class, interface
+                        or trait must be separated
```

Comment:
:+1:  updated in https://github.com/SpacePossum/PHP-CS-Fixer/commit/b8a5aa7281ee58ff6fa12a1922dddc2217cb6df6